components/FunkyArrow.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+
+const FunkyArrow = styled.a`

```

Comment:
Should also use the `Link` from the next package.
There is probably a guide out there for using it with styled-components